arm-storsimple8000series/2017-06-01/swagger/storsimple.json

```diff
@@ -245,13 +245,1142 @@
+        "description": "Creates or Updates an access control record.",
+        "operationId": "AccessControlRecords_CreateOrUpdate",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "accessControlRecordName",
+            "in": "path",
+            "description": "The name of the access control record.",
+            "required": true,
+            "type": "string",
+            "x-ms-skip-url-encoding": true
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "description": "The access control record to be added or updated.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/AccessControlRecord"
+            }
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/managerNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully created or updated the access control record.",
+            "schema": {
+              "$ref": "#/definitions/AccessControlRecord"
+            }
+          },
+          "202": {
+            "description": "Accepted the request to create or update the access control record."
+          }
+        },
+        "deprecated": false,
+        "x-ms-long-running-operation": true
+      },
+      "delete": {
+        "tags": [
+          "AccessControlRecords"
+        ],
+        "description": "Deletes the access control record.",
+        "operationId": "AccessControlRecords_Delete",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "accessControlRecordName",
+            "in": "path",
+            "description": "The name of the access control record to delete.",
+            "required": true,
+            "type": "string",
+            "x-ms-skip-url-encoding": true
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/managerNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted the request to delete the access control record."
+          },
+          "204": {
+            "description": "Successfully deleted the access control record."
+          }
+        },
+        "deprecated": false,
```

Comment:
@sergey-shandar, is it fine, if we address this is next PR, as these will require some changes in swashbuckle(to prevent this property to be added by default to each operation).